src/modules/organisation/api/organisation.uc.ts

```diff
@@ -34,4 +36,37 @@ export class OrganisationUc {
         }
         throw result.error;
     }
+
+    public async findAll(organisationDto: FindOrganisationDto): Promise<Paged<OrganisationResponse>> {
+        const organisationDo: OrganisationDo<false> = this.mapper.map(
+            organisationDto,
+            FindOrganisationDto,
+            OrganisationDo,
+        );
+        const result: Paged<OrganisationDo<true>> = await this.organisationService.findAllOrganizations(
+            organisationDo,
+            organisationDto.offset,
+            organisationDto.limit,
+        );
+
+        if (result.total === 0) {
+            return {
+                total: result.total,
+                offset: result.offset,
+                limit: result.limit,
+                items: [],
+            };
+        }
+
+        const organisations: OrganisationResponse[] = result.items.map((organisation: OrganisationDo<true>) =>
```

Comment:
Use `this.mapper.mapArray(...)` instead.

Reply:
✅